### PR TITLE
Clockwork Slabs can now focus on a specific component type to produce

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Clockwork Slabs can now focus on a specific component type to produce.
/:cl:

### _**EVEN IN THE MIDDLE OF WINTER, FEATURES BLOSSOM BENEATH THE SNOW**_

### CODE REVIEW

**Comment by ma44** [comment]
AAAAA CLOCKWORK BUFF

**Comment by KorPhaeron** [comment]
The one time I was a clock cultist other than a scarab we spent half a round waiting for a single type of component so we could build a tinker thing so this is a good change

**Comment by Armhulen** [comment]
i'm really glad that all the components are all equal

need for vanguard: heal, nothing else

need for replicant: IT'S IN EVERY SINGLE REQUIRED ITEM

**Comment by KingPhilipIII** [comment]
Maybe just buffing the generation rate of Replicant alloy specifically would work. That is by far the most needed and usually least available component.

**Comment by Magikarpet** [comment]
This is necessary but honestly it's the only thing that prevents decent clock cultists from snowballing and insta GGing the station with a speedcult 5-10 minutes in like a blood cult, which may be the intention.

**Comment by Armhulen** [comment]
oh yeah

this will basically make cuck cults win every time

**Comment by ChangelingRain** [comment]
Replicant usage is a problem but ¯\\\_(ツ)_/¯ you can't just make proselytizers not use it or use any component, you end up in a rabbit hole of progressively worse choices and problems.

**Comment by ChangelingRain** [comment]
Hey, maybe I can get oranges to merge this too!

**Comment by coiax** [comment]
...

File: tgui/src/interfaces/clockwork_slab.ract [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <ui-display>
   <ui-button action='toggle'>{{data.recollection ? "Recital":"Recollection"}}</ui-button>
+  <ui-button action='component'>Target Component: {{{data.target_comp}}}</ui-button>
```
Comment (coiax) [inline tgui/src/interfaces/clockwork_slab.ract]: mix and match of tgui and input...

Comment (ChangelingRain) [inline tgui/src/interfaces/clockwork_slab.ract]: There's no really good way to handle input in tgui, as far as I know; dropdown menus are a fucking myth for liars that don't exist.

**Pull request merged**